clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -786,6 +808,26 @@ private void handleAbortedSends(List<ClientResponse> responses) {
         abortedSends.clear();
     }
 
+    /**
+     * Handle socket channel connection timeout. The timeout will hit iff a connection
+     * stays at the ConnectionState.CONNECTING state longer than the timeout value,
+     * as indicated by ClusterConnectionStates.NodeConnectionState.
+     *
+     * @param responses The list of responses to update
+     * @param now The current time
+     */
+    private void handleTimeOutConnections(List<ClientResponse> responses, long now) {
+        Set<String> connectingNodes = connectionStates.connectingNodes();
+        for (String nodeId: connectingNodes) {
+            if (connectionStates.isConnectionSetupTimeout(nodeId, now)) {
+                // close connection to the node
```

Comment:
comment unnecessary since it is obvious from the line below

Reply:
Line deleted.